coreos-with-fleet-multivm/azuredeploy.json

```diff
@@ -0,0 +1,230 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
+            "metadata": { 
+                 "description": "Location of resources" 
+            } 
+        },
+        "newStorageAccountName": {
+            "type": "string",
+            "defaultValue": "storageaccount",
+            "metadata": { 
+                 "description": "Name of the storage account for VM OS Disks"
+            } 
+        },
+        "vmNamePrefix": {
```

Comment:
The storage account name is a global name that is not scoped to a resource group. I don't think the ARM template language has a way to generate a random name (although I'm not keen on random names due to the difficulty they provide in other contexts).